connector/connect/common/src/main/scala/org/apache/spark/sql/connect/client/GrpcExceptionConverter.scala

```diff
@@ -107,7 +107,7 @@ private[client] object GrpcExceptionConverter extends JsonUtils {
   private def toThrowable(ex: StatusRuntimeException): Throwable = {
     val status = StatusProto.fromThrowable(ex)
 
-    val fallbackEx = new SparkException(status.getMessage, ex.getCause)
+    val fallbackEx = new SparkException(ex.toString, ex.getCause)
```

Comment:
How was this before/after?

Reply:
Updated the description to include before/after. Please take a look